### ISSUE TITLE
Completion of Prim modules

### ISSUE THREAD

**Issue by milesfrain** [nwolverson/purescript-language-server#70]
Not sure if this is a duplicate, or if this is even the right repo to report this issue.

I noticed that automatic import with completion in my IDE (vscode) doesn't work on `Prim` classes. For example, I can type `getBody`, press tab, and then this line is inserted:
```
import Node.Express.Request (getBody)
```
But typing `RowToList` doesn't bring up any autocomplete options, and I need to manually add this import:
```
import Prim.RowList (class RowToList)
```

**Comment by AndrewBrownK** [comment]
This might be because of the following

https://pursuit.purescript.org/builtins/docs/Prim.Row
> The Prim.Row module is embedded in the PureScript compiler. **Unlike Prim, it is not imported implicitly.** It contains automatically solved type classes for working with row types.

It makes enough sense to not implicitly import these modules, but that doesn't mean they should be precluded from suggested imports when you start typing them out

**Comment by kritzcreek** [comment]
I had to check, but this should at least be possible with the current `purs ide`. Here's Emacs:
![rowtolist](https://user-images.githubusercontent.com/6189397/79547727-9e71e180-8094-11ea-91f1-8905d5b25ed1.gif)

**Comment by nwolverson** [comment]
OK, this is because for whatever reason we filter module completion by the set of available modules. Not sure why we explicitly filter including all modules in this case, but ignoring that historic oddness for now, I've included the current `Prim.*` modules in the completion for 

1. import statements
2. completion of identifiers in those modules

**Comment by nwolverson** [comment]
Apparently this was implemented in f886e8ee4887f160fff9c1ece553af2a43efd06c - not sure why I didn't close this at the time
